### PR TITLE
New layout for `time_tests`

### PR DESCRIPTION
**Pull request by vurusovs**

### CODE REVIEW

File: tests/time_tests/README.md [hunk 1]
```diff
@@ -21,6 +21,6 @@ cmake .. -DInferenceEngineDeveloperPackage_DIR=$(realpath ../../../build) && mak
 
 2. Run test:
 ``` bash
-./run_executable.py ../../../bin/intel64/Release/timetest_infer -m model.xml -d CPU
+./run_timtest.py ../../../bin/intel64/Release/timetest_infer -m model.xml -d CPU
```
Comment (asomsiko) [inline tests/time_tests/README.md:24]: typo - run_timEtest.py

File: tests/time_tests/scripts/run_timetest.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 """
-This script runs TimeTests executable several times to aggregate
+This script runs timetest executable several times to aggregate
```
Comment (asomsiko) [inline tests/time_tests/scripts/run_timetest.py:6]: and aggregate

File: tests/time_tests/README.md (hunk 1 above)
Comment (asomsiko) [reply tests/time_tests/README.md:24]: it should be ./scripts/run_timetest.py

**Review by asomsiko — CHANGES_REQUESTED** [review]

File: tests/time_tests/README.md (hunk 1 above)
Comment (vurusovs) [reply tests/time_tests/README.md]: Thanks! Fixed

File: tests/time_tests/scripts/run_timetest.py (hunk 2 above)
Comment (vurusovs) [reply tests/time_tests/scripts/run_timetest.py]: Fixed

**Review by asomsiko — APPROVED** [review]

**Pull request merged**